Laser_Lockers/FS6.json

```diff
@@ -517,7 +520,7 @@
                   "string": null,
                   "severity_on_failure": 2,
                   "if_disconnected": 2,
-                  "value": false,
+                  "value": 1.0,
```

Comment:
This doesn't appear to be the same check as the other FS configs

Reply:
that was my test to see if those values actually changed. turns out they don't so this value is irrelevant for now